<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="CenterEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Center">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>窗位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TransferButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度变换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查看原图</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PathEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/Users/richardchen0714/Documents/Enhancement/</string>
   </property>
  </widget>
  <widget class="QLabel" name="Path">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WidthEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Width">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>窗宽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SharpenAndTransferButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>锐化后灰度变换</string>
   </property>
  </widget>
  <widget class="QLabel" name="KernelSize1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>KernelSize1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KernelSizeEdit1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AlphaEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Alpha">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>平滑参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="SharpenParameter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>锐化参数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
